src/index.html

```diff
@@ -11,12 +11,113 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav-item">
+            <a
+              href="/apple"
+              class="nav_link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="/samsung"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="/smartphones"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="/laptops&computers"
```

Comment:
The 'href' attribute value '/laptops&computers' contains an ampersand ('&'), which should be encoded as '%26' to ensure proper URL encoding. Consider changing it to '/laptops%26computers' to avoid potential issues with URL parsing.